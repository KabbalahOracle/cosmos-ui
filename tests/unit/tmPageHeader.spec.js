import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import TmPageHeader from "@/components/tmPageHeader/tmPageHeader.vue"

describe("TmPageHeader.vue", () => {
  let wrapper
  beforeEach(async () => {
    const title = "Title"
    const subtitle = "Subtitle"
    wrapper = shallowMount(TmPageHeader, {
      propsData: { title, subtitle },
      slots: {
        default: "<div>slot</div>"
      }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
