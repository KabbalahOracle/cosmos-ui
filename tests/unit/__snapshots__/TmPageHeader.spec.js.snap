// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmPageHeader.vue has the expected html structure 1`] = `
<header class="tm-page-header tm-page-header-default">
  <div class="tm-page-header-container">
    <h1 class="title">
      Title
    </h1>
    <p class="subtitle">
      Subtitle
    </p>
    <div>
      slot
    </div>
  </div>
</header>
`;
